internal/cluster/cluster_test.go

```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/submariner-io/submariner-operator/internal/cluster"
 )
 
-var _ = Describe("IsValidID", func() {
+var _ = Describe("TestClusterIDs", func() {
```

Comment:
You should add new tests instead of piggy backing on these ones, as the functionality of the two methods isnt the same